Analyse fails with: Content Package Converter Exception Entry is outside of the target directory

since release 1.4.18 our maven builds fails in the analyse setup with this exception. it worked fine with 1.4.16 and below.
```
[ERROR] Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:1.4.18:project-analyse (aem-analyser) on project mycompany.myprojectgroup.myproject1.config-definition: Content Package Converter Exception Entry is outside of the target directory -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:1.4.18:project-analyse (aem-analyser) on project mycompany.myprojectgroup.myproject1.config-definition: Content Package Converter Exception Entry is outside of the target directory
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoExecutionException: Content Package Converter Exception Entry is outside of the target directory
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:294)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:259)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.io.IOException: Entry is outside of the target directory
    at org.apache.sling.feature.cpconverter.handlers.slinginitialcontent.SlingInitialContentBundleEntryMetaDataCollector.extractFile (SlingInitialContentBundleEntryMetaDataCollector.java:128)
    at org.apache.sling.feature.cpconverter.handlers.slinginitialcontent.SlingInitialContentBundleEntryMetaDataCollector.collectFromContextAndWriteTmpFiles (SlingInitialContentBundleEntryMetaDataCollector.java:103)
    at org.apache.sling.feature.cpconverter.handlers.slinginitialcontent.BundleSlingInitialContentExtractor.extract (BundleSlingInitialContentExtractor.java:80)
    at org.apache.sling.feature.cpconverter.handlers.SlingInitialContentBundleHandler.processBundleInputStream (SlingInitialContentBundleHandler.java:53)
    at org.apache.sling.feature.cpconverter.handlers.BundleEntryHandler.handle (BundleEntryHandler.java:141)
    at org.apache.sling.feature.cpconverter.vltpkg.RecollectorVaultPackageScanner.onFile (RecollectorVaultPackageScanner.java:61)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:112)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:104)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:91)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.firstPass (ContentPackage2FeatureModelConverter.java:288)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.convert (ContentPackage2FeatureModelConverter.java:267)
    at com.adobe.aem.analyser.AemPackageConverter.convert (AemPackageConverter.java:180)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:289)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:259)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
```

the content package that is validated *is* inside the target directory.

steps to reproduce the problem:
1. create a new AEM project using this archetype:
```

mvn org.apache.maven.plugins:maven-archetype-plugin:3.1.0:generate -DinteractiveMode=false \
  -DarchetypeGroupId=io.wcm.maven.archetypes \
  -DarchetypeArtifactId=io.wcm.maven.archetypes.aem \
  -DarchetypeVersion=3.6.4 \
  -DprojectName=myproject1 \
  -DgroupId=mycompany.myprojectgroup \
  -DartifactId=mycompany.myprojectgroup.myproject1 \
  -Dversion=1.0.0-SNAPSHOT \
  -Dpackage=mycompany.myprojectgroup.myproject1 \
  -DpackageGroupName=mycompany \
  -DaemAuthorPort=4502 \
  -DaemPublishPort=4503 \
  -DoptionJavaVersion=11 \
  -DoptionAemVersion=cloud \
  -DoptionAemServicePack=n \
  -DoptionAemServicePackAPI=n \
  -DoptionSlingInitialContentBundle=n \
  -DoptionEditableTemplates=y \
  -DoptionMultiBundleLayout=n \
  -DoptionContextAwareConfig=y \
  -DoptionWcmioHandler=n \
  -DoptionSlingModelsLatest=n \
  -DoptionContextAwareConfigLatest=n \
  -DoptionAcsCommons=n \
  -DoptionFrontend=n \
  -DoptionNodeJsPlugin=n \
  -DoptionIntegrationTests=n
```
2. in `config-definition/pom.xml` add `<version>1.4.18</version>` to the definition of `aemanalyser-maven-plugin`
3. build the project with `mvn clean install`

--- a/aem-project-tool/pom.xml
+++ b/aem-project-tool/pom.xml
@@ -49,7 +49,7 @@ governing permissions and limitations under the License.
               <outputDirectory>${project.build.directory}/classes</outputDirectory>
               <overWriteReleases>false</overWriteReleases>
               <overWriteSnapshots>true</overWriteSnapshots>
-              <includeArtifactIds>aemanalyser-core,org.osgi.framework,org.osgi.dto,org.osgi.resource,commons-lang,org.apache.sling.feature,org.apache.sling.feature.extension.apiregions,geronimo-json_1.1_spec,johnzon-core,org.apache.felix.cm.json,org.osgi.util.converter,org.osgi.util.function,slf4j-api,slf4j-simple</includeArtifactIds>
+              <includeArtifactIds>aemanalyser-core,org.osgi.framework,org.osgi.dto,org.osgi.resource,commons-lang,org.apache.sling.feature,org.apache.sling.feature.extension.apiregions,org.apache.sling.repoinit.parser,geronimo-json_1.1_spec,johnzon-core,org.apache.felix.cm.json,org.osgi.util.converter,org.osgi.util.function,slf4j-api,slf4j-simple</includeArtifactIds>
             </configuration>
           </execution>
         </executions>
@@ -142,9 +142,13 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.feature.cpconverter</artifactId>
-      <version>1.1.18</version>
+      <version>1.1.20</version>
     </dependency>
-
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.repoinit.parser</artifactId>
+      <version>1.7.0</version>
+    </dependency>
     <dependency>
         <groupId>org.apache.felix</groupId>
         <artifactId>org.apache.felix.configadmin.plugin.interpolation</artifactId>

--- a/aemanalyser-core/pom.xml
+++ b/aemanalyser-core/pom.xml
@@ -72,7 +72,7 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.feature.cpconverter</artifactId>
-      <version>1.1.18</version>
+      <version>1.1.20</version>
     </dependency>
 
     <dependency>

--- a/aemanalyser-maven-plugin/pom.xml
+++ b/aemanalyser-maven-plugin/pom.xml
@@ -182,7 +182,7 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.repoinit.parser</artifactId>
-      <version>1.6.14</version>
+      <version>1.7.0</version>
     </dependency>
     <dependency>
       <groupId>javax.jcr</groupId>
@@ -224,7 +224,7 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.feature.cpconverter</artifactId>
-      <version>1.1.18</version>
+      <version>1.1.20</version>
     </dependency>
     <dependency>
       <groupId>org.apache.felix</groupId>

